internal/organizer/db/companies_part3.go

```diff
@@ -4,43 +4,1138 @@ import "github.com/readytotouch/readytotouch/internal/domain"
 
 func companiesPart3() []domain.CompanyProfile {
 	return []domain.CompanyProfile{
-		// Template
-		//{
-		//	ID:   0,  // system
-		//	Type: "", // system
-		//	Name: "",
-		//	URL:  "",
-		//	LinkedInProfile: domain.LinkedInProfile{
-		//		ID:       0,
-		//		Alias:    "",
-		//		Name:     "",
-		//		Verified: false,
-		//	},
-		//	GitHubProfile: domain.GitHubProfile{
-		//		Login:             "",
-		//		GoRepositoryCount: 0,
-		//		Verified:          false,
-		//	},
-		//	GlassdoorProfile: domain.GlassdoorProfile{
-		//		OverviewURL: "",
-		//		ReviewsURL:  "",
-		//		Verified:    false,
-		//	},
-		//	OttaProfileSlug:   "",
-		//	YouTubeChannelURL: "",
-		//	GoMainLanguage:    false,
-		//	Vacancies: domain.Vacancies{
-		//		domain.Go:      nil,
-		//		domain.Rust:    nil,
-		//		domain.Zig:     nil,
-		//		domain.Scala:   nil,
-		//		domain.Elixir:  nil,
-		//		domain.Clojure: nil,
-		//		domain.Haskell: nil,
-		//	},
-		//	ShortDescription:          "",
-		//	Industries:                []domain.Industry{},
-		//	HasEmployeesFromCountries: []domain.Country{},
-		//},
+		// Pro5.ai  - AI-powered platform
```

Comment:
We ignore recruitment platforms because it’s difficult to determine if they’re hiring for themselves or for clients